lib/elm-test.js

```diff
@@ -285,10 +281,11 @@ function runElmTest() {
   checkNodeVersion();
 
   if (args._[0] == "init") {
-    var cmdArgs = Init.init();
-    var cmd = [pathToElmPackage, "install", "--yes"].concat(cmdArgs).join(" ");
-
-    child_process.execSync(cmd, { stdio: "inherit", cwd: Init.elmPackageDir });
+    var packagesToInstall = Init.init();
+    if (packagesToInstall.length > 0) {
+      var cmd = [pathToElm, "install"].concat(packagesToInstall).join(" ");
+      child_process.execSync(cmd, { stdio: "inherit", cwd: Init.elmPackageDir });
+    }
 
     process.exit(0);
```

Comment:
Should there be some feedback in this case?

Reply:
You mean if there are no packages to install?   I don't think it's necessary because `Init.init()` will also be printing other output.  But also, I didn't dig into the `elm-test init` code much; just enough to make the tests pass.